### DISCUSSION TITLE
Set environment variables on pods created by k8sRunLauncher

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#8515] [Q&A]
Hi everyone,
I am using k8sRunLauncher so that pipelines run in separate pods. But, these pods are missing some of the environment variables that are available to the repository pods. How do I add those variables to the pipeline run pods?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1641667223115500?thread_ts=1641667223.115500&cid=C01U954MEER).*

**Comment by yuhan** [comment]
You can set environment variables on the pods created by the run launcher using the fields here: https://github.com/dagster-io/dagster/blob/0.15.0/helm/dagster/values.yaml#L454-L461
